spec/unit/lib/temporal/workflow/dispatcher_spec.rb

```diff
@@ -83,6 +86,7 @@
         expect(handler_1).to have_received(:call).ordered
         expect(handler_4).to have_received(:call).ordered
         expect(handler_6).to have_received(:call).ordered
+        expect(handler_7).to have_received(:call).ordered
```

Comment:
Before this change, the sequence here would have been 1, 4, 7, 6 instead because targeted handlers were invoked before wildcard handlers